dev/tasks/tasks.yml

```diff
@@ -193,6 +196,7 @@ tasks:
       unicode_width: 16
       test_docker_images:
         - python:3.7-slim
+        - centos/python-37-centos7
```

Comment:
This image doesn't exists, that's why the corresponding crossbow task fails, removing it.